database/db_map.py

```diff
@@ -43,4 +41,74 @@ def get_receive_stations(self, rcv_station_dict):
                                                     categories: list(int),
                                                     items: list(int)}, ...}
         """
-        raise NotImplementedError
+        response = self.db.get("receiver_stations", None)
+        self.checking(rcv_station_dict, response)
+        return response
+
+    def checking(self, user_station, stations):
+        """
```

Comment:
Better to check inside the function only one element if it suits user_info. Iterate over stations in `get_receive_stations(...)`